index.html

```diff
@@ -1,11 +1,167 @@
 <!DOCTYPE html>
-<html lang="en">
+<html lang="en" class="page">
+
 <head>
   <meta charset="UTF-8">
   <meta name="viewport" content="width=device-width, initial-scale=1.0">
-  <title>Document</title>
+  <link rel="preconnect" href="https://fonts.googleapis.com">
+  <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+  <link
+    href="https://fonts.googleapis.com/css2?family=Raleway:wght@700&family=Roboto:ital,wght@0,400;0,500;0,700;0,900;1,400;1,500;1,700;1,900&display=swap"
+    rel="stylesheet">
+  <link rel="stylesheet" href="css/styles.css">
+  <title>Homework 2</title>
 </head>
-<body>
-  
+
+<body class="page__body">
+  <header>
+    <a class="logo" href="index.html">
+      <span class="logo__web">Web</span>
+      <span class="logo__studio logo__studio--black">Studio</span>
+    </a>
+    <ul class="navigation">
+      <li>
+        <a class="navigation__link" href="#">Студія</a>
+      </li>
+      <li>
+        <a class="navigation__link" href="portfolio.html">Портфоліо</a>
+      </li>
+      <li>
+        <a class="navigation__link" href="#">Контакти</a>
+      </li>
+    </ul>
+    <div class="contacts">
+      <a class="contacts__link" href="mailto:info@devstudio.com">
+        info@devstudio.com
+      </a>
+      <a class="contacts__link" href="tel:+380961111111">
+        +38 096 111 11 11
+      </a>
+    </div>
+  </header>
+
+  <main>
+    <section class="hero page__section">
+      <h1 class="hero__main-title">
+        Ефективні рішення для вашого бізнесу
+      </h1>
+      <button class="hero__button">Замовити послугу</button>
+    </section>
+
+    <section class="features">
+      <ul class="features__list">
+        <li class="features__item feature">
+          <h3 class="features__title">
+            Увага до деталей
+          </h3>
+          <p class="features__text">
+            Идейні міркування, і навіть початок повсякденної роботи з формування позиції.
+          </p>
+        </li>
+        <li>
```

Comment:
this `<li>` (and the following) is an elememt of `features__list`, and it should also have a class name as the first `<li>` - `features__item feature`


```suggestion
        <li class="feature__item">
```